src/posh-git.psd1

```diff
@@ -1,10 +1,10 @@
 @{
 
 # Script module or binary module file associated with this manifest.
-ModuleToProcess = 'posh-git.psm1'
+RootModule = 'posh-git.psm1'
 
 # Version number of this module.
-ModuleVersion = '0.7.0'
+ModuleVersion = '1.0.0'
```

Comment:
The way I handle that scenario is to import by path e.g:
```powershell
Import-Module $Home\GitHub\dahlbyk\posh-git\src\posh-git.psd1
```
Development is in the works for PowerShellGet/PSGallery to support prereleases.  I'm hoping this problem will get solved in the coming months.  https://github.com/PowerShell/PowerShell-RFC/issues/42